packages/core/src/domain/telemetry/telemetry.spec.ts

```diff
@@ -31,6 +42,39 @@ describe('telemetry', () => {
     expect(notifySpy).toHaveBeenCalledTimes(1)
   })
 
+  describe('addTelemetryConfiguration', () => {
+    afterEach(() => {
+      resetExperimentalFeatures()
+    })
+
+    it('should notify configuration when sampled and ff is enabled', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`notify configuration` seems a bit odd to me, what about something like `collects configuration` or `notify with configuration` instead?